migrations/versions/f80726457a72_.py

```diff
@@ -0,0 +1,28 @@
+"""Add Controlled_access field to genomic_file
+
+Revision ID: f80726457a72
```

Comment:
I think you should be able to combine these two migration script into one. You could do: 

```
git reset origin/master migrations
rm migrations/versions/1c790b83f611_.py migrations/versions/f80726457a72_.py
# Make sure your dataservice containers are running, and if not run:
docker-compose up -d
# Generate new migration script
flask db migrate
``` 
That should generate a new migration script. Make sure you update the docstring to include both the field changes